editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -697,22 +700,10 @@ export const ComponentSectionInner = betterReactMemo(
             return <ParseErrorControl parseError={rootParseError} />
           },
           (rootParseSuccess) => {
-            const propNames = Object.keys(rootParseSuccess)
-
-            // TODO FIX ME
-            if (Math.random() > 0) {
```

Comment:
OMG :sweat_smile:

Reply:
WTF :D

Reply:
Wat.

Reply:
It was slipped in 2 weeks ago apparently :D